aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -702,43 +709,54 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         ellipsisSpinRight.setAnimationListener(
                 object : Animation.AnimationListener {
                     override fun onAnimationEnd(animation: Animation) {
-                        buttonEllipsisCollapse.visibility = View.VISIBLE
-                        buttonEllipsisExpand.visibility = View.GONE
+                        buttonEllipsisCollapsed.visibility = View.VISIBLE
+                        buttonEllipsisExpanded.visibility = View.GONE
                     }
 
                     override fun onAnimationRepeat(animation: Animation) {
                     }
 
                     override fun onAnimationStart(animation: Animation) {
                         layoutExpanded.visibility = View.VISIBLE
-                        layoutExpanded.startAnimation(layoutExpandedTranslateInRight)
+                        //in rtl mode the scrollview will scroll to "end" when layoutExpanded becomes visible
+                        //keep hard focus on media button to avoid it
+                        toolbarScrolView.requestChildFocus(buttonMediaCollapsed, buttonMediaCollapsed)
+                        layoutExpanded.startAnimation(layoutExpandedTranslateInEnd)
                     }
                 }
         )
     }
 
+    fun scrollToBeginingOfToolbar() {
+        if (TextUtilsCompat.getLayoutDirectionFromLocale(Locale.getDefault()) == ViewCompat.LAYOUT_DIRECTION_LTR
```

Comment:
We might consider adding a comment explaining that HorizontalScrollView on older android versions doesn't support RtL layouts.

Reply:
Good call! I added a comment about RTL support to this method and to the toolbar itself.